src/bynder-js-sdk.js

```diff
@@ -478,6 +478,31 @@ export default class Bynder {
         return request.send();
     }
 
+    /**
+     * Modify new metaproperty with the information provided.
+     * @see {@link https://bynder.docs.apiary.io/#reference/metaproperties/specific-metaproperty-operations/modify-metaproperty|API Call}
+     * @param {Object} object={} - An object containing the data of the metaproperty.
```

Comment:
Since `object.id` is mandatory, we can add to the docs as well:
`     * @param {String} queryObject.id - The id of the desired metaproperty.`

Reply:
Thanks! Added this.